charts/keycloak/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: keycloak
-version: 10.1.0
+version: 10.1.1
```

```suggestion
version: 10.2.1
```
The version need to increase with each release. I just merged another PR so there is no fault on your side.